lib/smart_proxy_remote_execution_ssh/command_action.rb

```diff
@@ -42,7 +42,7 @@ def rescue_strategy_for_self
     def command
       @command ||= Dispatcher::Command.new(:id               => input[:task_id],
                                            :host             => input[:hostname],
-                                           :ssh_user         => 'root',
+                                           :ssh_user         => input[:ssh_user] || 'root',
```

Comment:
What we are currently looking is a way how to determine the effective user (via sudo) at the server side, but it could be similar to the remote user. Let's chat on monday about that